payjoin/src/io.rs

```diff
@@ -17,10 +18,10 @@ pub async fn fetch_ohttp_keys(
     ohttp_relay: impl IntoUrl,
     payjoin_directory: impl IntoUrl,
 ) -> Result<OhttpKeys, Error> {
-    let ohttp_keys_url = payjoin_directory.into_url()?.join("/ohttp-keys")?;
+    let ohttp_keys_url = payjoin_directory.into_url()?.join("/.well-known/ohttp-gateway")?;
     let proxy = Proxy::all(ohttp_relay.into_url()?.as_str())?;
     let client = Client::builder().proxy(proxy).build()?;
-    let res = client.get(ohttp_keys_url).send().await?;
+    let res = client.get(ohttp_keys_url).header(ACCEPT, "application/ohttp-keys").send().await?;
```

Comment:
You mention that this isn't checked anywhere in the directory. Is there a plan to require that? Seems to not make a huge difference.